modules/oadp-configuring-restore-pvc-datamover-restore.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/configuring-backup-restore-pvc-datamover.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-restore-pvc_{context}"]
+= Configuring `restorePVC` for Data Mover restore
+
+A `restorePVC` is an intermediate PVC that is used to write data during the Data Mover restore operation.
+
+You can configure the `restorePVC` in the `DataProtectionApplication` (DPA) by using the `ignoreDelayBinding` field. Setting the `ignoreDelayBinding` field to true allows the restore operation to ignore the `WaitForFirstConsumer` binding mode. The data movement restore operation then creates the restore pod and provisions the associated volume to an arbitrary node. 
```

Comment:
```suggestion
You can configure the `restorePVC` in the `DataProtectionApplication` (DPA) object by using the `ignoreDelayBinding` field. Setting the `ignoreDelayBinding` field to `true` allows the restore operation to ignore the `WaitForFirstConsumer` binding mode. The data movement restore operation then creates the restore pod and provisions the associated volume to an arbitrary node. 
```
Should _data movement restore operation_ be _Data Mover restore operation_ for consistency?